encfs/FileUtils.cpp

```diff
@@ -931,8 +933,9 @@ static bool selectChainedIV() {
   // xgroup(setup)
   return boolDefaultYes(
       _("Enable filename initialization vector chaining?\n"
-        "This makes filename encoding dependent on the complete path, \n"
-        "rather then encoding each path element individually."));
```

Comment:
I'd rather track feature incompatibilities in the man page instead of translation strings.